src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -403,9 +404,39 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             return "";
         }
 
-        boolean isContainsNonEmptyResponse = false; // we will only show stats if there is at least one nonempty response
-
         Map<String, Integer> answerFrequency = new LinkedHashMap<String, Integer>();
+        int numChoicesSelected = getQuestionResultStatistics(responses, answerFrequency);
+        if (numChoicesSelected == -1) {
+            return "";
+        } else {
```

Comment:
What I'm saying is
```java
if(x){
    return a;
} else {
   return b;
}
```
is same as
```java
if(x){
    return a;
}

return b;
```

The 2nd option avoids nesting the primary path of the method.